src/main/resources/anniv5Resources/localization/eng/rimworldpack/Cardstrings.json

```diff
@@ -38,14 +38,12 @@
   },
   "${ModID}:BurningPassion": {
     "NAME": "Burning Passion",
-    "DESCRIPTION": "Double the first buff you gain each turn."
+    "DESCRIPTION": "Exhaust your hand. NL Gain [E] \u00a0and draw 1 card for each card Exhausted. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "Exhaust any number of cards. Gain [E] \u00a0and draw 1 card for each card Exhausted. NL Exhaust."
   },
   "${ModID}:TaintedWeapon": {
     "NAME": "Tainted Weapon",
-    "DESCRIPTION": "Deal !D! damage. NL Startup: Put this card in a random monster.",
-    "EXTENDED_DESCRIPTION": [
-      "Deal !D! damage."
-    ]
+    "DESCRIPTION": "Deal !D! damage. NL Put this card in the enemy if they are not dead."
```

Comment:
Might make sense to use similar "if this doesn't kill" wording to Kill Thirst above. Combined with the comment I made about Kill Thirst's wording, that would make Tainted Weapon "If this doesn't kill, put it in the enemy."

It's the same idea in both cases (an effect that happens if it doesn't kill), so nice to use the same wording.